ci/gpu/build.sh

```diff
@@ -45,15 +45,18 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda create install "cudf>=0.9" "dask-cudf>=0.9" "cudatoolkit=$CUDA_REL" \
-              "dask>=2.3.0" "distributed>=2.3.2" "numpy>=1.16" "cupy>=6.2.0"
+conda install "cudf>=0.10" "dask-cudf>=0.10" "cudatoolkit=$CUDA_REL" \
+              "dask>=2.3.0" "distributed>=2.3.2" "numpy>=1.16" -c rapidsai-nightly
 
 # needed for asynccontextmanager in py36
 conda install -c conda-forge "async_generator" "automake" "libtool" \
                               "cmake" "automake" "autoconf" "cython" \
                               "pytest" "pkg-config" "pytest-asyncio"
 
 
+# install cupy
+conda install -c rapidsai -c nvidia "cupy>=6.3.0"
```

Comment:
Is there a reason it needs to be CuPy 6.3.0? Or would CuPy 6.2.0 work?

Reply:
cupy 6.20 should also be fine.  I'll get a PR in late today

Reply:
Added PR ( https://github.com/rapidsai/ucx-py/pull/239 ).